Traceback (most recent call last):
  File "/home/runner/.cache/pypoetry/virtualenvs/sds-adv-ml-t6Z6UgPC-py3.9/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/runner/.cache/pypoetry/virtualenvs/sds-adv-ml-t6Z6UgPC-py3.9/lib/python3.9/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/runner/.cache/pypoetry/virtualenvs/sds-adv-ml-t6Z6UgPC-py3.9/lib/python3.9/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/runner/.cache/pypoetry/virtualenvs/sds-adv-ml-t6Z6UgPC-py3.9/lib/python3.9/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/runner/.cache/pypoetry/virtualenvs/sds-adv-ml-t6Z6UgPC-py3.9/lib/python3.9/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/runner/.cache/pypoetry/virtualenvs/sds-adv-ml-t6Z6UgPC-py3.9/lib/python3.9/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
batch_size = 32

transform = transforms.Compose(
    [
        transforms.Grayscale(),
        transforms.RandomAffine(degrees=[-90, 90], scale=[0.8, 1.1]),
        transforms.ToTensor(),
        transforms.ConvertImageDtype(torch.float32),
    ]
)

dataset = HiraganaDataset(dataroot="hiragana73", transform=transform)
train_data, test_data = torch.utils.data.random_split(dataset, [60000, 20000])
train_loader = DataLoader(train_data, batch_size=batch_size, shuffle=True)
test_loader = DataLoader(test_data, batch_size=batch_size)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [0;32mIn[5], line 12[0m
[1;32m      1[0m batch_size [38;5;241m=[39m [38;5;241m32[39m
[1;32m      3[0m transform [38;5;241m=[39m transforms[38;5;241m.[39mCompose(
[1;32m      4[0m     [
[1;32m      5[0m         transforms[38;5;241m.[39mGrayscale(),
[0;32m   (...)[0m
[1;32m      9[0m     ]
[1;32m     10[0m )
[0;32m---> 12[0m dataset [38;5;241m=[39m [43mHiraganaDataset[49m[43m([49m[43mdataroot[49m[38;5;241;43m=[39;49m[38;5;124;43m"[39;49m[38;5;124;43mhiragana73[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[43mtransform[49m[38;5;241;43m=[39;49m[43mtransform[49m[43m)[49m
[1;32m     13[0m train_data, test_data [38;5;241m=[39m torch[38;5;241m.[39mutils[38;5;241m.[39mdata[38;5;241m.[39mrandom_split(dataset, [[38;5;241m60000[39m, [38;5;241m20000[39m])
[1;32m     14[0m train_loader [38;5;241m=[39m DataLoader(train_data, batch_size[38;5;241m=[39mbatch_size, shuffle[38;5;241m=[39m[38;5;28;01mTrue[39;00m)

Cell [0;32mIn[3], line 8[0m, in [0;36mHiraganaDataset.__init__[0;34m(self, dataroot, transform)[0m
[1;32m      5[0m [38;5;28mself[39m[38;5;241m.[39mdataroot [38;5;241m=[39m dataroot
[1;32m      6[0m [38;5;28mself[39m[38;5;241m.[39mtransform [38;5;241m=[39m transform
[0;32m----> 8[0m folders [38;5;241m=[39m [38;5;28msorted[39m([d [38;5;28;01mfor[39;00m d [38;5;129;01min[39;00m [43mos[49m[38;5;241;43m.[39;49m[43mlistdir[49m[43m([49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mdataroot[49m[43m)[49m])
[1;32m      9[0m chars [38;5;241m=[39m [[38;5;28mchr[39m([38;5;28mint[39m(d[38;5;241m.[39mreplace([38;5;124m"[39m[38;5;124mU[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124m0x[39m[38;5;124m"[39m), [38;5;241m16[39m)) [38;5;28;01mfor[39;00m d [38;5;129;01min[39;00m folders]
[1;32m     10[0m n_chars [38;5;241m=[39m [38;5;28mlen[39m(chars)

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'hiragana73'

